src/settings-ui/Settings.UI/SettingsXAML/Views/EnvironmentVariablesPage.xaml

```diff
@@ -27,17 +27,13 @@
                 <controls:SettingsGroup x:Uid="EnvironmentVariables_Activation_GroupSettings" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
                     <controls1:SettingsCard
                         x:Uid="EnvironmentVariables_LaunchButtonControl"
-                        ActionIcon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily},
-                                                 FontSize=14,
-                                                 Glyph=&#xE8A7;}"
+                        ActionIcon="{ui:FontIcon Glyph=&#xE8A7;}"
```

Comment:
This icon falls back to MDL2. Does that mean there's something wrong in the implementation?

https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/6bfe8a5d21feb8818a8048b9fc41a84a70af8b0f/Microsoft.Toolkit.Uwp.UI/Extensions/Markup/FontIconExtension.cs#L30-L38